src/db/models/ApplicationsModel.py

```diff
@@ -9,6 +9,14 @@
 class ApplicationsModel(BaseModel):
     TableName = "applications"
     Model = {}
+    count_query = sql.SQL("""SELECT COUNT("score") FROM %s WHERE "score">0""" % TableName)
+    total_count_query = sql.SQL("""SELECT COUNT("score") FROM %s """ % TableName) 
```

Comment:
Can we use `COUNT(*)` for these two queries? `score` is initialized to 0 so they should both return the same result

Reply:
the first one is counting how many are finished I think, look at the where

Reply:
Your queries work as is, but I'm asking whether `COUNT("score")` is needed and whether we can substitute `COUNT(*)` for it. It would only make a difference if some rows have a null `score` column, which shouldn't be the case. If they both return the same result, I'm more inclined to go with `COUNT(*)`.